### DISCUSSION TITLE
if i use hyper-express should i remove my uwebsockets package and use the one inbuilt in hyper-express?

### DISCUSSION THREAD

**Discussion by ggg7125** [kartikk221/hyper-express#60] [General]
i am very inexperienced. i have a project where i was already using uwebsockets before i even knew about hyper-express. but now that i am going to use hyper-express in the project (it is a game) would it be best to remove my uwebsockets from the package.json and just use the uwebsockets built into hyper-express for my game's socket communications?

**Comment by kartikk221** [comment]
Yes, since uWebsockets.js is a dependency of hyper-express, as long as you have hyper-express installed, you can uninstall uWebsockets.js.
